polaris-react/src/components/TopBar/TopBar.scss

```diff
@@ -138,8 +160,18 @@ $icon-size: 20px;
     // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
     margin-left: calc((100% - #{$page-max-width}) / 2);
   }
+
+  #{$se23} & {
+    @media #{$p-breakpoints-md-down} {
```

Comment:
typically we encourage using `up` instead of down https://polaris.shopify.com/tokens/breakpoints#sass-variables